Logos/varmour-logo.svg

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Please move the logo to solutions folder, it should be at "Azure-Sentinel/Solutions/vArmour/Data Connectors/Logo".

Reply:
Relocated logo to  Azure-Sentinel/Solutions/vArmour/Data Connectors/Logo